app/controllers/albums.js

```diff
@@ -0,0 +1,22 @@
+const albumsService = require('../services/albums'),
+  logger = require('../logger');
+
+exports.albumsGET = (req, res, next) =>
+  albumsService
```

Comment:
I would add a logger here to log when this process start. Without this logger, if the controller fails, you won't log anything.